### PR TITLE
Relation Type model definition

### PR DESCRIPTION
**Pull request by Matthew-Wise**
Added the model definitions of the RelationType model. Most of the text taken from the Summary comments in the code :)

### CODE REVIEW

File: Reference/Management/Models/RelationType.md [hunk 1]
```diff
@@ -17,6 +17,47 @@ All samples in this document will require the following using statement:
...
+	return relationType.ParentObjectType;
+    
+ ### .ChildObjectType
+  Gets or sets the Childs object type id as `Guid`
+  
+	// Given a `RelationService` object get RelationType by its Id and return IsBidirectional
+	var relationType = relationService.GetRelationTypeById(1234);
+	return relationType.ChildObjectType;
+    
+## Methods
```
Comment (jmayntzhusen) [inline Reference/Management/Models/RelationType.md]: Is there a reason you left a `Methods` headline with no content under it? 🙂

Comment (Matthew-Wise) [reply Reference/Management/Models/RelationType.md]: No I should of just removed it

**Comment by dampee** [comment]
The idea is that we are not starting to copy paste all properties in the docs. The auto-generated content should be auto-generated.  Otherwise, we have a massive copy-paste job.
Having a description at the top is good, as well as examples, guides or tips on how and when to (not) use it.

While we like to thank you for the effort @Matthew-Wise, I think we should partially reject your PR.  I hope you understand the "we do not like copy paste"-vision. Do you see a chance to update your PR accordingly?

**Comment by Matthew-Wise** [comment]
@dampee  I agree on the copy paste content, I think some of it is unavoidable as adding further explanation to some properties would be unnecessary. I will however update some that need a bit more explaining.

I'll take a look at the meta/yaml as well.

**Comment by Matthew-Wise** [comment]
I also noticed on the Content one it includes properties from "Entity" so I will add those. 

Or should an Entity page be created and that inheritance made clear? So we avoid maintaining Entity in so many places.

**Comment by dampee** [comment]
@Matthew-Wise 
Sorry for the waiting, there were a few issues with the markdown, but i've fixed them for you!
Thanks a lot for the contribution

**Pull request merged**